lib/ansible/plugins/vars/host_group_vars.py

```diff
@@ -47,6 +47,12 @@
         ini:
           - section: yaml_valid_extensions
             key: defaults
+            deprecated:
```

Comment:
It doesn't seem like this actually worked - can we just remove the entry?

Reply:
it does technically work,
```ini
 [yaml_valid_extensions]
defaults = 
```
tempted to just remove, but deprecation is the safe choice